components/Order/order_transitions.rst

```diff
@@ -0,0 +1,35 @@
+.. _component_order_order-transitions:
+
+OrderTransitions
+================
+
+There are following order's transitions by default:
+
++------------------+------------+
+| Related constant | Transition |
++==================+============+
+| SYLIUS_CREATE    | create     |
++------------------+------------+
+| SYLIUS_RELEASE   | release    |
++------------------+------------+
+| SYLIUS_CONFIRM   | confirm    |
++------------------+------------+
+| SYLIUS_SHIP      | ship       |
++------------------+------------+
+| SYLIUS_ABANDON   | abandon    |
++------------------+------------+
+| SYLIUS_CANCEL    | cancel     |
++------------------+------------+
+| SYLIUS_RETURN    | return     |
++------------------+------------+
+
+There is also the default graph name included:
+
++------------------+--------------+
+| Related constant | Name         |
++==================+==============+
+| GRAPH            | sylius_order |
++------------------+--------------+
+
+.. note::
+    All of above transitions and the graph are constant fields in the **OrderTransitions** class.
```

Comment:
Actually (ref. my previous comment), I think we should have a single chapter ``state_machine`` with (suggested) states and transitions defined there. No need for having them separated. Agree?